### ISSUE TITLE
Tracking of allowed-address-pairs broken with l2pop/arp responder and LinuxBridge/VXLAN

### ISSUE THREAD

**Issue by d34dh0r53** [rcbops/rpc-openstack#874]
This is to track the progress of [`1445089`](https://bugs.launchpad.net/neutron/+bug/1445089) a bug in Neutron which affects ML2/LinuxBridge and VXLAN networks `allowed-address-pairs` functionality.

We should push reviews on the patch and work aggressively to get this backported into liberty.  At this time nothing in RPC-O will need to change, but I want this to be a reminder to go poke this issue periodically.

- [x] Master
- [x] Liberty-12.2
- [x] Liberty-12.1

**Comment by d34dh0r53** [comment]
The fix for this has merged into master, the liberty backport is here: [`288050`](https://review.openstack.org/#/c/288050/)

**Comment by prometheanfire** [comment]
liberty has merged, just waiting on a sha update from OSA now I think

**Comment by b3rnard0** [comment]
Waiting for SHA bump. @git-harry any idea of which one will include this change?

**Comment by prometheanfire** [comment]
should be in #1054 #1055 and #1056
